### ISSUE TITLE
Change the default number of results?

### ISSUE THREAD

**Issue by tupton** [tupton/alfred-chrome-history#9]
Even though only 9 results are visible in Alfred, you can scroll or arrow down for more if more are available. Is it worth making more (15? 20? 50?) available, or would more results be less relevant and clutter things?

**Comment by Jyqft** [comment]
Yes, I think more results would be better—9 seems too limiting. Maybe 50's good?

**Comment by tupton** [comment]
Thanks for the feedback. Do you enter your query and then scroll through the results? I find that I just try to narrow down my query, but I understand that not everyone can or wants to do that, and I don't think it'll hurt anything to have more results.

I'll test out a higher number – 50 sounds good; it's a max not a hard limit – and see how it goes.

**Comment by Jyqft** [comment]
I sometimes do that, yeah.

**Comment by ydnikolaev** [comment]
Hey from 2020
@tupton thanks for great workflow, it is still works (Catalina, Alfred 4, Chrome 81 )

I changed number of results to 30 (and made some little changes to db query) and it works great

Small tip: I made "Argument Optional" and a shortcode Cmd + Y (which is active only when focus on Alfred)
 
so now I can Open Alfred > hit Cmd + Y and immidiatly get list of last used links
just like a native chrome history search
perfect

one more time, thanks for such a great work

**Comment by tupton** [comment]
@ydnikolaev thanks for the feedback! Pull requests are welcome – if you want to share your modifications and we can figure out how to make them opt-in or backwards compatible, I'd love to get them in.

**Comment by jerryyang0611** [comment]
> Hey from 2020
> @tupton thanks for great workflow, it is still works (Catalina, Alfred 4, Chrome 81 )
> 
> I changed number of results to 30 (and made some little changes to db query) and it works great
> 
> Small tip: I made "Argument Optional" and a shortcode Cmd + Y (which is active only when focus on Alfred)
> 
> so now I can Open Alfred > hit Cmd + Y and immidiatly get list of last used links
> just like a native chrome history search
> perfect
> 
> one more time, thanks for such a great work

how do you change the number of result to 30? Thanks in advance.
